readme.md

```diff
@@ -8,15 +11,21 @@ npm install validate-date --save--dev
 
 ## Usage
 
+The function accepts two parameters: a date string and a "boolean" parameter. By default "boolean"=false and the function returns strings (see below). If "boolean" is set to true, the function returns "true" for validated dates and "false" for incorrect ones.
+
 ```js
 
 var validateDate = require("validate-date");
 
 validateDate('99/10/2020'); //returns Invalid Date
 validateDate('02/29/2001'); //returns Invalid Date. Leap year check verified
 validateDate('02022000'); // returns Invalid Format
+validateDate('02022000', boolean=false); // returns Invalid Format
 validateDate(''); // returns Invalid Format
 validateDate('02/02/2001'); //returns Valid Date
+validateDate('11/01/1991', boolean=true); //returns true
+validateDate('02022000', boolean=true); // returns false
```

Comment:
Let's update these to the one below Please make sure to preserve the bold text :)

validateDate('99/10/2020'); //returns Invalid Date
validateDate('02/29/2001'); //returns Invalid Date. Leap year check verified
validateDate('02022000'); // returns Invalid Format
validateDate(''); // returns Invalid Format
validateDate('02/02/2001'); //returns Valid Date

validateDate('99/10/2020’, booleanResponse=true); // returns false
validateDate('02/29/2001’,  booleanResponse=true); //returns false. Leap year check verified
validateDate('02022000’, booleanResponse=true); // returns false
validateDate(‘’, booleanResponse=true); // returns false
validateDate('02/02/2001', booleanResponse=true); //returns true

The **booleanResponse** param is false by default.
validateDate('99/10/2020’, booleanResponse=**false**); //returns Invalid Date
validateDate('99/10/2020’, booleanResponse=**true**); //returns false

Reply:
Great =)

Reply:
Not sure how to add bold inside the `js` part of the code...

Reply:
Not in the js. I meant in the readme.md file. Just enclose any given word by double-asterisk(** **) to make the word bold